recipes/parg/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
```

Comment:
```suggestion
project(test_package C)
```

Reply:
I am curious, does it have any actual effect?

Reply:
Yes, it doesn't try to find or call a C++ compiler or linker (nice to properly test C++ library with C API and a C test package). Here it's not very important.